inputs/values/dockers.json

```diff
@@ -29,4 +29,4 @@
   "gq_recalibrator_docker": "us.gcr.io/broad-dsde-methods/markw/gatk:mw-tb-form-sv-filter-training-data-899360a",
   "str": "us.gcr.io/broad-dsde-methods/gatk-sv/str:2023-05-23-v0.27.3-beta-e537bdd6",
   "denovo": "us.gcr.io/broad-dsde-methods/gatk-sv/denovo:2024-06-04-v0.28.5-beta-a8dfecba"
-}
\ No newline at end of file
+}
```

Comment:
You should revert this trivial change as well. You can be sure by using `git checkout main inputs/values/dockers.json`

Reply:
This should be fixed. Thank you!